winlocalprocessspawner/win_utils.py

```diff
@@ -68,6 +143,9 @@ def _execute_child(self, args, executable, preexec_fn, close_fds,
             comspec = os.environ.get("COMSPEC", "cmd.exe")
             args = '{} /c "{}"'.format(comspec, args)
 
+        sid, _ = win32security.GetTokenInformation(self._token, win32security.TokenUser)
+        setup_sacl(sid)
```

Comment:
the spawner can be used without the winauthenticator, in which case self._token will be None and this will fail. Can you add a "if token:" block around this code?